src/playground/tabs/SettingsTab.tsx

```diff
@@ -961,11 +961,18 @@ function LinterSettings({
 						aria-describedby="lint-rules-description"
 						name="lint-rules"
 						disabled={!enabledLinting}
-						value={lintRules ?? LintRules.Recommended}
-						onChange={(e) => setLintRules(e.target.value as LintRules)}
+						value={lintRules}
+						onChange={(e) =>
+							setLintRules(e.target.value as keyof typeof LINT_RULES)
+						}
 					>
-						<option value={LintRules.Recommended}>Recommended</option>
-						<option value={LintRules.All}>All</option>
+						{Object.values(LINT_RULES).map((value) => {
```

Comment:
It would be nice if `LINT_RULES` was sorted by name. It would be even better if you could search the list from the selector, but I won't block this PR for that.

Reply:
Yes, that was intentional. I'm not blocking on adding search. I am blocking on the sorting. Not having these be sorted is a really bad UX because we have 300+ rules.

Reply:
Ah ok, it wasn't clear

Reply:
Done